config/features/flags.go

```diff
@@ -229,6 +234,7 @@ var BeaconChainFlags = append(deprecatedBeaconFlags, append(deprecatedFlags, []c
 	DisableRegistrationCache,
 	EnableLightClient,
 	BlobSaveFsync,
+	EnableQUIC,
```

Comment:
can you add this to our dev flags too ? So that any user running with `--dev`is also running this

Reply:
Fixed in 1f0d84d0094f5dd58743e4316fa4be4043679614.